src/resources/js/routes.js

```diff
@@ -2,14 +2,19 @@ import Vue from 'vue';
 import VueRouter from 'vue-router';
 
 import Hello from './components/Hello';
+import Presentation from './components/Presentation';
 
 Vue.use(VueRouter);
 
 export const router = new VueRouter({
   mode: 'history',
-    routes: [{
-        path: '/hello',
-        name: 'Hello',
-        component: Hello
-    }],
```

Comment:
Does this need to be removed in this PR ? Not sure

Reply:
this doesn't have to be in my PR since I remove Hello.vue,
Must I revert it?

Reply:
but my headache will start when I will merge

Reply:
Seems fair enough, good luck with merging !